client/signup/config/flows-pure.js

```diff
@@ -388,6 +388,7 @@ export function generateFlows( {
 				? [
 						// Add or replace new steps here
 						'user',
+						'difm-page-picker',
```

Comment:
Since both `difm-options` and `social-profiles` are merged, we can have set `difm-page-picker` as the last step in the flow.

Reply:
Fixed here [3579bda](https://github.com/Automattic/wp-calypso/pull/61790/commits/3579bdadabbeafcbee5fda5431fe69562ebdb7c8)